### ISSUE TITLE
reserved identifier violation

### ISSUE THREAD

**Issue by elfring** [Microsoft/cpprestsdk#14]
I would like to point out that identifiers like "[`_CASA_HTTP_CLIENT_H`](https://github.com/Microsoft/cpprestsdk/blob/5b525c47fb0f11c89b69263a1e13feb02e9abcb6/Release/include/cpprest/http_client.h#L27)" and "[`_Value`](https://github.com/Microsoft/cpprestsdk/blob/6839a4638101f927a62b547cbdd7900469ac7039/Release/include/cpprest/json.h#L44)" [do not fit](https://www.securecoding.cert.org/confluence/display/cplusplus/DCL51-CPP.+Do+not+declare+or+define+a+reserved+identifier "Do not use identifiers that are reserved for the compiler implementation.") to the expected naming convention of the C++ language standard.
Would you like to adjust your selection for unique names?

**Comment by ras0219-msft** [comment]
Yes, these identifiers are in violation of the expected naming convention for non-standard-library code. We would love to have them changed, but so far other issues have been more pressing.

I'm sure a PR for this would be gratefully accepted.
